docs/app/pages/components/components-sections/search/logical-expression-builder/logical-expression-builder.component.html

```diff
@@ -0,0 +1,331 @@
+            Contains a number picker. It expects a value of type <code>number</code>. The <code>data</code> can contain
+            the following properties:
+        </p>
+
+        <ul>
+            <li>
+                <code>min: number</code> - The smallest number that can be entered in the date picker.
+            </li>
+            <li>
+                <code>max: number</code> - The largest number that can be entered in the date picker.
+            </li>
+            <li>
+                <code>validateFunction: (value: number) => boolean</code> - If specified, this
+                function is used to validate the value entered by the user.
+            </li>
+        </ul>
+    </tr>
+
+    <tr uxd-api-property name="SelectInputComponent">
+        <p>
+            Contains a select input. It expects a value of type <code>string[]</code>, containing the <code>name</code>
+            of one or more options. The options for the select input can be provided in using the <code>options</code>
+            property of the <code>data</code>. Each option needs the
+            following properties:
+        </p>
+
+        <ul>
+            <li>
+                <code>name: string</code> - A unique identifier for the option.
+            </li>
+            <li>
+                <code>label: string</code> - The text that is displayed in the UI.
+            </li>
+            <li>
+                <code>icon?: string</code> - Optionally, the name of a <code>ux-icon</code> that is displayed for this
+                option.
+            </li>
+        </ul>
+
+        <p>Additionally, a function to validate the entered value can be provided.</p>
+
+        <ul>
+            <li>
+                <code>validateFunction: (value: string[]) => boolean</code> - If specified, this
+                function is used to validate the value entered by the user.
+            </li>
+        </ul>
+    </tr>
+
+    <tr uxd-api-property name="TextInputComponent">
+        <p>
+            Contains a simple text input, expecting a value of type <code>string</code>. The <code>data</code> can contain
+            the following properties:
+        </p>
+
+        <ul>
+            <li>
+                <code>validateFunction: (value: string) => boolean</code> - If specified, this
+                function is used to validate the value entered by the user.
+            </li>
+        </ul>
+    </tr>
+</uxd-api-properties>
+
+<h4>Custom Input Components</h4>
+
+<p>
+    In addition to the predefined Input Components, it is also possible to built custom input components. These must
+    conform with the following interfaces:
+</p>
+
+<uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="value" type="any">
+        Contains the value of the respective condition.
+    </tr>
+
+    <tr uxd-api-property name="data" type="any">
+        Contains the data for the specific <code>field</code> defined in the fields definition. E.g., it can be used
```

Comment:
Perhaps something like:

"The configuration options for the custom component, e.g. the available options in a select dropdown. This is defined in the `fields` input of `ux-logical-expression-builder`."